pkg/maven/maven.go

```diff
@@ -103,6 +127,130 @@ func Evaluate(pomFile, expression string, command mavenExecRunner) (string, erro
 	return value, nil
 }
 
+// InstallFile installs a maven artifact and its pom into the local maven repository.
+// If "file" is empty, only the pom is installed.
+// "pomFile" may not be empty.
+func InstallFile(file, pomFile string, command mavenExecRunner, utils mavenUtils, projectSettingsFile, globalSettingsFile, m2Path string) error {
+	if len(pomFile) == 0 {
+		return fmt.Errorf("pomFile can't be empty")
+	}
+
+	var defines []string
+	if len(file) > 0 {
+		defines = append(defines, "-Dfile="+file)
+	} else {
+		defines = append(defines, "-Dfile="+pomFile)
+	}
+	defines = append(defines, "-DpomFile="+pomFile)
+	mavenOptionsInstall := ExecuteOptions{
+		Flags:                       []string{},
+		Goals:                       []string{"install:install-file"},
+		Defines:                     defines,
+		PomPath:                     pomFile,
+		ProjectSettingsFile:         projectSettingsFile,
+		GlobalSettingsFile:          globalSettingsFile,
+		M2Path:                      m2Path,
+		LogSuccessfulMavenTransfers: false,
```

Comment:
Can probably strip Flags and LogSuccessfulMavenTransfers fields.

Reply:
what is the default value of LogSuccessfulMavenTransfers? I'd like it to be false in this case.

Reply:
The default value is `false`, meaning you do not see log messages about downloaded dependencies by default.